README.md

```diff
@@ -1 +1,17 @@
 to do list
+
```

Comment:
readme.md można ładnie formatować 
https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/